src/raylib.h

```diff
@@ -806,7 +806,8 @@ typedef enum {
 typedef enum {
     BLEND_ALPHA = 0,        // Blend textures considering alpha (default)
     BLEND_ADDITIVE,         // Blend textures adding colors
-    BLEND_MULTIPLIED        // Blend textures multiplying colors
+    BLEND_MULTIPLIED,       // Blend textures multiplying colors
+    BLEND_SET               // Blend textures by setting colors
```

Comment:
Would you mind renaming it to `BLEND_ADD_COLORS`? It sounds a bit more descriptive to me.

Reply:
Done